### ISSUE TITLE
Hours showing with decimal, minutes empty in time picker

### ISSUE THREAD

**Issue by otoivanen** [nielsfaber/scheduler-card#779]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

v3.2.12

### Bug description

When adding a schedule, the time picker shows minutes as decimals in Hour -section and the minutes stay empty.

When increasing minutes from the up and down arrows, only decimals change in hours -section (and the decimals don't fit in the area if value is longer than two digits).

![kuva](https://github.com/nielsfaber/scheduler-card/assets/55193463/67a750c6-9191-4aaa-af32-b378c42c2c2f)

When googling found only one similar case which was also Finnish user. Maybe localization related?

### Steps to reproduce

Not sure how to reproduce, this is how it has been since installation on Finnish setup.

**Comment by otoivanen** [comment]
Just noticed, that if I select the before/after sunrise mode, then minutes are showing correctly when setting the offset. Not working when giving exact time.

**Comment by otoivanen** [comment]
Tried also changing locale to english in Chrome, but only the character changed from comma to dot in the hour picker.

**Comment by nielsfaber** [comment]
Could you try setting the HA language to English and see if the issue disappears?
Perhaps it is related to Finnish locale.

**Comment by otoivanen** [comment]
> Could you try setting the HA language to English and see if the issue disappears?
> Perhaps it is related to Finnish locale.

Yes, I did some investogation and the reason was that HA has 00.00 time format in Finnish, but 00:00 in all other nordic countries.

I checked the card's javascript, and modified the getTimeParts function which splits on ":" - replaced it with "."  and now the picker works! Just testing if scheduling also works as supposed to.

Maybe the source could be updated to split time parts also on "."?

**Comment by nielsfaber** [comment]
Looks like you did all the work, thanks!
I was not aware of this different notation. This is definitely something that should be addressed in the code.
I will put it on my to-do list.

**Comment by Christoffer-da** [comment]
I have the same error with Danish locale-settings.

**Comment by mrmaceurope** [comment]
+1 As mention still an issue with Danish

**Comment by nielsfaber** [comment]
@mrmaceurope did you try with scheduler-card v4 ?

**Comment by mrmaceurope** [comment]
@nielsfaber would that be stable... and which version would you recommend (latest) ?

<img width="198" height="193" alt="Image" src="https://github.com/user-attachments/assets/4e1f1b1d-d101-4365-9112-1bcd8baa05fa" />

**Comment by nielsfaber** [comment]
@mrmaceurope I am not aware of any issues with the latest v4.0.0.b4 version. If you experience them, please let me know. You can also switch back to v3.2.15 later on, no information will get lost.

**Comment by mrmaceurope** [comment]
@nielsfaber works perfectly 👌🏼 thanks

**Comment by nielsfaber** [comment]
I am closing this issue since this problem does not occur in v4 of the card.
